assets/css/_route_ladder.scss

```diff
@@ -1,5 +1,6 @@
 .m-route-ladder {
   margin: 0 0.5rem;
+  text-align: center;
```

Comment:
Not sure if this is the line that's doing it or not, but the Reverse button has moved to the middle from where it should be on the left edge.

Reply:
Ah, I saw that but didn't register it as wrong. I'll center the SVG with `margin: auto` instead. Good catch.